lemarche/www/tenders/views.py

```diff
@@ -45,3 +48,34 @@ def get_initial(self):
 
     def get_success_message(self, cleaned_data, tender):
         return mark_safe(self.success_message.format(tender.title))
+
+
+class TenderListView(LoginRequiredMixin, ListView):
+    template_name = "tenders/list_tenders.html"
+    model = Tender
+    context_object_name = "tenders"
+    paginate_by = 10
+    paginator_class = Paginator
+
+    def get_queryset(self):
+        user = self.request.user
+        # user kind : buyer
+        if user.kind == User.KIND_BUYER:
+            queryset = Tender.objects.created_by_user(user)
+            # user kind : siae
+        elif user.kind == User.KIND_SIAE and user.siaes:
+            # TODO: manage many siaes
+            siae = user.siaes.first()
+            sectors = siae.sectors.all()
+            queryset = Tender.objects.in_sectors(sectors).find_in_perimeters(
+                post_code=siae.post_code, coords=siae.coords, department=siae.department, region=siae.region
+            )
+        else:
+            queryset = Tender.objects.all()
+        return queryset
+
+
+class TenderDetail(DetailView):
+    model = Tender
+    template_name = "tenders/detail_tender.html"
```

Comment:
```suggestion
    template_name = "tenders/detail.html"
```

plus simple bis :) (cf templates/siaes/detail.html)